### PR TITLE
RHDEVDOCS-2740 Restructure the Logging release notes to set the stage for upcoming 5.0.1 release.

### PR DESCRIPTION
**Pull request by rolfedh**
- For versions 4.7+
- https://issues.redhat.com/browse/RHDEVDOCS-2740
- Direct link to doc preview: https://deploy-preview-30094--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html
- Ready for peer review and merge (restructured content does not require SME review)

### CODE REVIEW

**Review by rolfedh — COMMENTED** [review]
@vimalk78 Please hold https://errata.devel.redhat.com/docs/show/70510 until this PR has been published.

File: logging/cluster-logging-release-notes.adoc [hunk 1]
```diff
@@ -8,235 +8,14 @@ toc::[]
...
-// {ProductName} release 5.0 is now available. The list of bug fixes that are included in the update is documented in the link:https://errata.devel.redhat.com/docs/show/66974[RHBA-2020:66974-04] advisory.
-
-// UNUSED IN THIS 5.0.0 RELEASE - KEEP THIS BOILERPLATE FOR FUTURE RELEASES
-// The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2020:5678[RHBA-2020:5678] advisory.
-//
-// Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
-//
-// link:https://access.redhat.com/solutions/<ARTICLE_ID>[{ProductName} 5.0.0 container image list]
+include::modules/cluster-logging-release-notes-5.0.0.adoc[leveloffset=+1]
+//include::modules/cluster-logging-release-notes-5.0.1.adoc[leveloffset=+1]
```
Comment (rolfedh) [inline logging/cluster-logging-release-notes.adoc:21]: @vikram-redhat  I've commented out the 5.0.1 RNs for now so we can merge the general changes now. Then I'll create another PR for the 5.0.1 changes.

File: logging/cluster-logging-release-notes.adoc [hunk 2]
```diff
@@ -8,235 +8,14 @@ toc::[]
 [id="openshift-logging-5-0-about-this-release"]
 == About this release
 
-(link:https://errata.devel.redhat.com/docs/show/66974[RHBA-2020:66974-04 Errata Advisory for Openshift Logging 5.0.0]) is now available. New features, changes, and known issues that pertain to {ProductName} 5.0 are included in this topic.
+The following advisories are available for {ProductName} 5.0:
+
+* link:https://errata.devel.redhat.com/docs/show/66974[RHBA-2020:66974-04 Errata Advisory for Openshift Logging 5.0.0]
```
Comment (rolfedh) [inline logging/cluster-logging-release-notes.adoc:13]: @vikram-redhat  I might want to move this into modules/cluster-logging-release-notes-5.0.0.adoc. What do you think?

File: logging/cluster-logging-release-notes.adoc [hunk 3]
```diff
@@ -8,235 +8,14 @@ toc::[]
...
 
 [id="openshift-logging-5-0-inclusive-language"]
 == Making open source more inclusive
 
 Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright’s message].
 
-[id="openshift-logging-5-0-new-features-and-enhancements"]
```
Comment (rolfedh) [inline logging/cluster-logging-release-notes.adoc:18]: @vikram-redhat  I pasted all this content to cluster-logging-release-notes-5.0.0.adoc

File: modules/cluster-logging-release-notes-5.0.0.adoc [hunk 4]
```diff
@@ -0,0 +1,230 @@
+[id="cluster-logging-release-notes-5-0-0"]
```
Comment (rolfedh) [inline modules/cluster-logging-release-notes-5.0.0.adoc:1]: @vikram-redhat  Here's all that pasted content - no changes (I think).

File: modules/cluster-logging-release-notes-5.0.1.adoc [hunk 5]
```diff
@@ -0,0 +1,9 @@
+[id="cluster-logging-release-notes-5-0-1"]
+= OpenShift Logging 5.0.1
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.1.adoc:2]: @vikram-redhat  This is the content I'm still sorting out with the logging team. When it's ready, I'll create a new PR that updates this file and uncomments the include so it appears in the docs.

**Comment by vikram-redhat** [comment]
Thanks @rolfedh - lgtm.

File: modules/cluster-logging-release-notes-5.0.1.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (vimalk78) [reply modules/cluster-logging-release-notes-5.0.1.adoc:2]: ok

File: modules/cluster-logging-release-notes-5.0.0.adoc [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+This release includes Red Hat Bug Advisory, link:https://errata.devel.redhat.com/docs/show/66974[RHBA-2020:66974-04 Errata Advisory for Openshift Logging 5.0.0].
+
+[id="openshift-logging-5-0-new-features-and-enhancements"]
+== New features and enhancements
+
+This release adds improvements related to the following concepts.
+
+[discrete]
+[id="ocp-4-7-cluster-logging-renamed-openshift-logging"]
+==== Cluster Logging becomes Red Hat OpenShift Logging
```
Comment (Preeticp) [inline modules/cluster-logging-release-notes-5.0.0.adoc:13]: Is there a reason why this uses heading level 4 and not 3 like the other features in this section?

File: modules/cluster-logging-release-notes-5.0.0.adoc [hunk 7]
```diff
@@ -0,0 +1,230 @@
...
+
+* Previously, logs were not sent to managed storage when legacy log forwarding was enabled. This happened because the internal generation of the `logforwarding` configuration improperly made a decision for  either `logforwarding` or legacy `logforwarding`. The current release fixes this issue:  Logs are sent to managed storage when the logstore is defined in the `clusterlogging` instance.  Additionally, logs are sent to legacy `logforwarding` when enabled regardless of whether a managed logstore is enabled or not. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1921263[*1921263*])
+
+* Previously, the Kibana log level was increased not to suppress instructions to delete indices that failed to migrate, which also caused the display of GET requests at the INFO level that contained the Kibana user's email address and OAuth token. The current release fixes this issue by masking these fields, so the Kibana logs do not display them. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925081[*BZ#1925081*])
+
+
+[id="openshift-logging-5-0-technology-preview"]
+== Technology Preview features
```
Comment (Preeticp) [inline modules/cluster-logging-release-notes-5.0.0.adoc:158]: Suggestion: Add this section after the New features and enhancements section and before the Bug Fixes section.

**Review by Preeticp — COMMENTED** [review]
Hi, Rolfe, overall looks great. I have left a couple of minor suggestions, PTAL.

File: modules/cluster-logging-release-notes-5.0.0.adoc (hunk 6 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.0.adoc:13]: Nice catch. Fixed it.

**Pull request merged**